docs/source/release_notes/release-notes.rst

```diff
@@ -75,6 +74,13 @@ develop
           See :ref:`oracle_table_mapping` for details on how table names are mapped.
           (`Pull Request <https://github.com/palantir/atlasdb/pull/1076>`__)
 
+    *    - |improved|
+         - Additional debugging available for those receiving 'name must be no longer than 1500 bytes' errors.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1117>`__)
+
+    *    - |breaking|
+         - Cell.validateNameValid is now private; consider Cell.isNameValid instead
```

Comment:
Then we _probably_ don't need to call it out in the release notes (but thanks for the addition). The additional debugging note is useful IMHO.